docs/src/pages/[platform]/connected-components/authenticator/configuration/loginMechanisms.web.mdx

```diff
@@ -4,25 +4,26 @@ import { InlineFilter } from '@/components/InlineFilter';
 
 <Alert role="none" heading="Zero Configuration">
 
-The Authenticator [automatically infers](/connected-components/authenticator#step-1-configure-backend) `loginMechanisms` from `amplify pull`,
+The Authenticator [automatically infers](/connected-components/authenticator#step-1-configure-backend) `loginMechanisms` from the current Amplify configuration, 
 but can be explicitly defined as seen below.
 
 </Alert>
 
-By default, the Authenticator creates new users in the Amazon Cognito UserPool based on a unique `username` using
-[`Auth signUp`](https://docs.amplify.aws/javascript/build-a-backend/auth/enable-sign-up/).
+By default, the Authenticator creates new users in the Amazon Cognito UserPool based on a unique `username`.
```

Comment:
Does it make sense to continue including this line here if the immediate example below is for "Email"?

Reply:
Seems like we should just add a note that signing in with username is not possible in Gen 2, will add

Reply:
Ended up updating the note here to be more clear, let me know what you think!